using_images/db_images/mysql.adoc

```diff
@@ -154,27 +154,50 @@ MySQL settings can be configured with the following environment variables:
-* `mysql-persistent` uses a persistent volume store for the database data which means the data will survive a Pod restart.  Using persistent volumes requires a persistent volume pool be defined in the OpenShift deployment.  Instructions for setting up the pool are located link:../../admin_guide/persistent_storage_nfs.html[here].
-
-
-You can find instructions for instantiating templates by following these link:../../dev_guide/templates.html#creating-resources-from-a-template[instructions].
-
-Once you have instantiated the service, you can copy the username, password, and database name environment variables into a DeploymentConfig for another component that intends to access the database.  That component can then access the database via the Service that was defined.
+OpenShift provides a link:../../dev_guide/templates.html[template] to make
+creating a new database service easy. The template provides parameter fields to
+define all the mandatory environment variables (user, password, database name,
+etc) with predefined defaults including auto-generation of password values. It
+will also define both a
+link:../../architecture/core_concepts/deployments.html#deployments-and-deployment-configurations[deployment
+configuration] and a
+link:../../architecture/core_concepts/pods_and_services.html#services[service].
+
+The MySQL templates should have been registered in the default *openshift*
+project by your cluster administrator during the
+link:../../admin_guide/install/first_steps.html[First Steps] setup process.
+There are two templates available:
+
+* `mysql-ephemeral` is for development or testing purposes only because it uses
+ephemeral storage for the database content. This means that if the database
+pod is restarted for any reason, such as the pod being moved to another node
+or the deployment configuration being updated and triggering a redeploy, all
+data will be lost.
+* `mysql-persistent` uses a persistent volume store for the database data which
+means the data will survive a pod restart. Using persistent volumes requires a
+persistent volume pool be defined in the OpenShift deployment. Cluster
+administrator instructions for setting up the pool are located
+link:../../admin_guide/persistent_storage_nfs.html[here].
+
+
+You can find instructions for instantiating templates by following these
+link:../../dev_guide/templates.html#creating-resources-from-a-template[instructions].
+
+Once you have instantiated the service, you can copy the user name, password,
+and database name environment variables into a deployment configuration for
+another component that intends to access the database. That component can then
+access the database via the service that was defined.
 
 [[using-mysql-replication]]
-== Using MySQL Master-Slave Replication
+== Using MySQL Replication
```

Comment:
Sure. I put it there because in MySQL the replication works slightly differently than in MongoDB. Also in MySQL you can have Master-Master replication which is not supported here.